bugwarrior/services/deck.py

```diff
@@ -0,0 +1,192 @@
+import datetime
+import logging
+
+import requests
+import typing_extensions
+from dateutil.tz import tzutc
+
+from bugwarrior import config
+from bugwarrior.services import IssueService, Issue, ServiceClient
+
+log = logging.getLogger(__name__)
+
+
+class NextcloudDeckConfig(config.ServiceConfig, prefix='deck'):
+    service: typing_extensions.Literal['deck']
+    base_uri: config.StrippedTrailingSlashUrl
+    username: str
+
+    # can be a password or an app-password
+    password: str
+
+    include_board_ids: config.ConfigList = config.ConfigList([])
+    exclude_board_ids: config.ConfigList = config.ConfigList([])
+
+    import_labels_as_tags: bool = False
+    label_template: str = '{{label}}'
+    only_if_assigned: str = None
```

Comment:
I believe you should be able to remove the `only_if_assigned` field without consequence. It is defined in `ServiceConfig` from which this class inherits.

Reply:
Correct, thanks!